Sources/NIOCore/BSDSocketAPI.swift

```diff
@@ -93,7 +93,7 @@ let SO_TIMESTAMP = CNIOLinux_SO_TIMESTAMP
 let SO_RCVTIMEO = CNIOLinux_SO_RCVTIMEO
 #endif
 
-public enum NIOBSDSocket {
+public enum NIOBSDSocket: Sendable{
```

Comment:
I don't think we can make a socket `Sendable` it is the same as `FileHandle`

Reply:
This is just a namespace.

Reply:
Ah good catch. Then my comments from below apply. Should we really make namespaces `Sendable`?

Reply:
I can't see any reason to do so.